internal/core/tx.py

```diff
@@ -52,5 +102,5 @@ def tx_broadcast(signed_file, gas, broadcast_mode="sync"):
     if broadcast_mode == "block":
         logging.info("Waiting for transaction for being broadcasted")
     command = f"""{DAEMON} tx broadcast {HOME}/{signed_file} --output json \
---chain-id {CHAINID} --gas {gas} --node {RPC} --broadcast-mode {broadcast_mode}"""
+--chain-id {CHAINID} --gas {gas}stake --node {RPC} --broadcast-mode {broadcast_mode}"""
```

Comment:
what is `stake` here?
```suggestion
--chain-id {CHAINID} --gas {gas} --node {RPC} --broadcast-mode {broadcast_mode}"""
```